Modules/Promotion/Http/Controllers/PromotionController.php

```diff
@@ -3,11 +3,13 @@
 namespace Modules\Promotion\Http\Controllers;
 
 use Illuminate\Http\JsonResponse;
-use Illuminate\Http\Request;
 use Illuminate\Routing\Controller;
+use Illuminate\Http\Request;
```

Comment:
Remove 
use Illuminate\Http\Request;
use Illuminate\Support\Facades\Auth;
we are not using them